### PR TITLE
[hi] Localize content/en/docs/reference/glossary/cluster-architect.md

### PR DESCRIPTION
**Pull request by shivanii1702**
Signed-off-by: shivani tiwari <shivanishinu05@gmail.com>
fixes #35600

### CODE REVIEW

**Review by bishal7679 — COMMENTED** [review]
@Shivaniiiiiiiiiii Thanks for raising this PR
but this PR doesn't contain any content
It's completely empty.

**Comment by NitishKumar06** [comment]
@Shivaniiiiiiiiiii Please look at the point that @vishakhanihore mendtioned.

Till that time, I'll be keeping this PR in hold.

/hold

**Comment by Shivaniiiiiiiiiii** [comment]
@bishal7679 please check it now

**Comment by bishal7679** [comment]
/remove hold

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: content/hi/docs/reference/glossary/cluster-architect.md
Comment (bishal7679) [inline content/hi/docs/reference/glossary/cluster-architect.md:7]:
```suggestion
 एक व्यक्ति जो बुनियादी ढांचे को रचना करता है जिसमें एक या एक से अधिक कुबेरनेट्स क्लस्टर शामिल होते हैं।

```
Wouldn't रचना be better than the literal translation of design?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: content/hi/docs/reference/glossary/cluster-architect.md
Comment (bishal7679) [inline content/hi/docs/reference/glossary/cluster-architect.md:13]:
```suggestion
एक व्यक्ति जो बुनियादी ढांचे को रचना करता है जिसमें एक या एक से अधिक कुबेरनेट्स क्लस्टर शामिल होते हैं।

```
same as above

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by bishal7679 — CHANGES_REQUESTED** [review]

**Comment by Shivaniiiiiiiiiii** [comment]
@bishal7679 fixed all required changes.

File: content/hi/docs/reference/glossary/cluster-architect.md
Comment (sftim) [inline content/hi/docs/reference/glossary/cluster-architect.md:2]: For this one, no need to include the original English term (IMO)
```suggestion
title: क्लस्टर आर्किटेक्ट
```

Comment (NitishKumar06) [reply content/hi/docs/reference/glossary/cluster-architect.md]: +1 for this

**Comment by NitishKumar06** [comment]
The changes are fine. The only thing that I'm concerned of is that 1 check is not successful.

**Comment by bishal7679** [comment]
@NitishKumar06 tide is not successful because k8s/website repo is freezed now for tomorrows 1.25 release
And not a single PR can be merged before that release


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: content/hi/docs/reference/glossary/cluster-architect.md
Comment (Babapool) [inline content/hi/docs/reference/glossary/cluster-architect.md:17]: Can this line be localized in a better way something along the lines of this?:
```suggestion
क्लस्टर आर्किटेक्ट्स वितरित सिस्टम के लिए सर्वोत्तम प्रथाओं से संबंधित हैं, उदाहरण के लिए उच्च उपलब्धता और सुरक्षा।
```
Thoughts, WDYT @bishal7679 @NitishKumar06 @Shivaniiiiiiiiiii ?

Comment (Shivaniiiiiiiiiii) [reply content/hi/docs/reference/glossary/cluster-architect.md]: @bishal7679 now check this

Comment (bishal7679) [reply content/hi/docs/reference/glossary/cluster-architect.md]: @Babapool good suggestion but localize system to प्रणाली
👍

Comment (Shivaniiiiiiiiiii) [reply content/hi/docs/reference/glossary/cluster-architect.md]: @bishal7679  should I change सिस्टम to प्रणाली ?

Comment (bishal7679) [reply content/hi/docs/reference/glossary/cluster-architect.md]: Yes

Comment (NitishKumar06) [reply content/hi/docs/reference/glossary/cluster-architect.md]: @Babapool Don't you think that both the lines hold different meaning?

Comment (bishal7679) [reply content/hi/docs/reference/glossary/cluster-architect.md]: @NitishKumar06 wouldn't it be better if deto localization for english is done for this line?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Comment (NitishKumar06) [reply content/hi/docs/reference/glossary/cluster-architect.md]: @bishal7679 That's what I also feel like! Adding text from our end might result in some errors. It'd be better if we follow the idea you mentioned. WDYT of this @rolfedh ?

Comment (bishal7679) [reply content/hi/docs/reference/glossary/cluster-architect.md]: @rolfedh is not a native hindi speaker @NitishKumar06 😄

Comment (NitishKumar06) [reply content/hi/docs/reference/glossary/cluster-architect.md]: @bishal7679 Are you able to open the slack? I'm not able to:(

Comment (Babapool) [reply content/hi/docs/reference/glossary/cluster-architect.md]: @bishal7679 @NitishKumar06 My idea while suggesting this change was to tell the readers that the Cluster architecture is concerned/associated with the best practices of high availability and security in a distributed system.

However I wasn't too sure on this suggestion, so I asked for other contributors for their thoughts before this suggestion was taken up. Thanks

**Comment by NitishKumar06** [comment]
Hey, @Shivaniiiiiiiiiii I've observed that you've raised multiple commits of the same title for this issue. It's good to see that you're contributing! I'd like to give you one suggestion (personal) which I generally follow while raising multiple commits for the same issue :

Try to add a number next to the commit message. For eg: Update cluster.md-01, Update cluster.md-02...This will help to understand more about what file changes have been changed in which commit no. 

I hope you understood this.
Thank you !Keep contributing:)

**Comment by bishal7679** [comment]
> Hey, @Shivaniiiiiiiiiii I've observed that you've raised multiple commits of the same title for this issue. It's good to see that you're contributing! I'd like to give you one suggestion (personal) which I generally follow while raising multiple commits for the same issue :
> 
> Try to add a number next to the commit message. For eg: Update cluster.md-01, Update cluster.md-02...This will help to understand more about what file changes have been changed in which commit no.
> 
> I hope you understood this. Thank you !Keep contributing:)

I do suggest to squash all the commit into one atlast 
WDYT?

**Comment by NitishKumar06** [comment]
@bishal7679 No idea! I prefer raising multiple commits though since I've never tried squashing commits

File: content/hi/docs/reference/glossary/cluster-architect.md
Comment (NitishKumar06) [reply content/hi/docs/reference/glossary/cluster-architect.md]: @Babapool Gotcha! I'm waiting for @rolfedh suggestion on the idea proposed by @bishal7679 regarding translating the deto English text to Hindi.

Comment (rolfedh) [reply content/hi/docs/reference/glossary/cluster-architect.md]: > @bishal7679 That's what I also feel like! Adding text from our end might result in some errors. It'd be better if we follow the idea you mentioned. WDYT of this @rolfedh ?

Normally, you would be able to search for the id, `cluster-architect`, to see the glossary content in-context.
However, `dev-1.22-hi.1` doesn't have a topic that uses it.

Comment (rolfedh) [reply content/hi/docs/reference/glossary/cluster-architect.md]: Glossary embeds in `*.md` files look like this:
```
* {{< glossary_tooltip text="Deployment" term_id="deployment" >}}
```

Comment (NitishKumar06) [reply content/hi/docs/reference/glossary/cluster-architect.md]: Thanks @rolfedh

**Comment by Shivaniiiiiiiiiii** [comment]
> Hey, @Shivaniiiiiiiiiii I've observed that you've raised multiple commits of the same title for this issue. It's good to see that you're contributing! I'd like to give you one suggestion (personal) which I generally follow while raising multiple commits for the same issue :
> 
> Try to add a number next to the commit message. For eg: Update cluster.md-01, Update cluster.md-02...This will help to understand more about what file changes have been changed in which commit no.
> 
> I hope you understood this. Thank you !Keep contributing:)

@NitishKumar06 okay Nitish actually I was not aware of this and I will keep that in mind. Thanks

**Comment by rolfedh** [comment]
Commit messages can communicate the types of changes you're making to your PR. Some PR reviewers might find this helpful. This [Kubernetes blogs topic](https://kubernetes.io/docs/contribute/new-content/blogs-case-studies/#technical-considerations-for-submitting-a-blog-post) has some recommendations on what a good commit message looks like. 
But keep in mind that only the **first** commit message appears in the localization branch commit history after your PR is merged. This is because we squash the commits in our PRs to only one commit: https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits

File: content/hi/docs/reference/glossary/cluster-architect.md
Comment (NitishKumar06) [reply content/hi/docs/reference/glossary/cluster-architect.md]: Are there any more changes that need to be implemented now? @bishal7679 @Shivaniiiiiiiiiii

**Comment by rolfedh** [comment]
Hi @shivanii1702!  Thanks for your contributions to the /hi localization branch! Would you change the target branch from  `dev-1.22-hi.1`  to `main`? Here are some screenshots showing the steps to take: https://kubernetes.slack.com/archives/CJ14B9BDJ/p1664112024896939

**Comment by NitishKumar06** [comment]
@shivanii1702 please take a look at what @rolfedh has suggested

**Comment by rolfedh** [comment]
Hi @shivanii1702. Thanks for working your way through all these comments. I promise it gets easier with practice! :smile: 
Please have a look at https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits

**Comment by rolfedh** [comment]
You can also ask on the [#kubernetes-docs-hi](https://kubernetes.slack.com/archives/CJ14B9BDJ) Slack channel for someone to walk you through this squashing process. It's a bit tricky until you get used to it.

**Comment by divya-mohan0209** [comment]
Also, you might need to sync your fork since it seems outdated.

**Comment by NitishKumar06** [comment]
@shivanii1702 Ping

**Comment by divya-mohan0209** [comment]
@shivanii1702 I will be closing this PR since we do not have a response from your side for quite some time. If you wish to continue working on it, please reopen it and check the comments above.

/close

**Pull request closed without merging**
